pkg/generate/git/git.go

```diff
@@ -21,13 +21,14 @@ import (
 // - ssh
 func ParseRepository(s string) (*url.URL, error) {
 	uri, err := url.Parse(s)
-	if err != nil {
-		return nil, err
+	if err == nil && uri.Scheme != "" {
```

Comment:
I've added some negative tests.

If url.Parse returns an error, then the input will be interpreted using alternative git syntax. The error will be returned from the second step if the input is invalid.

But behaviour is changed: `example.com:3000` was an invalid string, now it is `ssh://example.com/3000` as I think it should be.